### DISCUSSION TITLE
请问怎么关闭CEF的右键菜单

### DISCUSSION THREAD

**Discussion by ZhenZY** [CefView/QCefView#150] [Q&A]
有没有通过简单的配置方式关闭下面这个功能。但是不可以关闭输入框里的右键菜单

![wecom-temp-2e7d46ebeb746a7cbbdae54460d3aa70](https://user-images.githubusercontent.com/25266953/191208135-4b430ec9-d605-4b81-a46e-233cb1955da5.jpg)

**Comment by tishion** [comment]
https://github.com/CefView/QCefView/pull/124
参考

**Comment by ZhenZY** [comment]
您好，我是姜丽珍，您的邮件我已收到，我将尽快回复您，谢谢。
